docker-compose.yml

```diff
@@ -4,6 +4,11 @@ services:
     container_name: palworld-dedicated-server
     image: jammsen/palworld-dedicated-server:latest
     restart: unless-stopped
+    logging:
+      driver: "local"
+      options:
+        max-size: "100m"
```

Comment:
Pretty sure 10MB is fairly okay, even 15MB is massive, 100 feels a bit overloaded, doesnt it?

Reply:
Sure, can reduce that.